docs/starters.yml

```
@@ -2619,3 +2619,52 @@
     - Styling with SCSS
     - Offline support
     - Web App Manifest
+- url: https://cosmicjs.com/apps/gatsby-agency-portfolio/demo
+  repo: https://github.com/cosmicjs/gatsby-agency-portfolio
+  description: Static Webpage for displaying your agencies skills and past work.  Implements 4 sections for displaying information about your company, A home page, information about services, projects, and the people in your organization.
+  tags:
+    - Landing Page
+    - Blog
+    - Portfolio
+    - Cosmic JS
+    - React
+    - Node JS
+  features:
+    - Static Site Generator
+    - Home
+    - Services
+    - Projects
+    - People
+- url: https://cosmicjs.com/apps/gatsby-localization-app-starter/demo
+  repo: https://github.com/cosmicjs/gatsby-localization-app-starter
+  description: A localized Gatsby starter application powered by Cosmic JS.
+  tags:
+    - React
+    - Node JS
+    - Cosmic JS
+    - Localization
+    - Web Dev
+  features:
+    - Gatsby localization starter app
+- url: https://cosmicjs.com/apps/gatsby-docs/demo
+  repo: https://github.com/cosmicjs/gatsby-docs-app
+  description: Be able to view and create documentation using Gatsby and Cosmic JS. Leveraging the speed and high powered APIs of the Gatsby framework and the simplicity and scalability of Cosmic JS.
+  tags:
+    - React
+    - Node JS
+    - Cosmic JS
+    - Docs
```

```suggestion
    - Docs
    + Documentation
```
Documentation is already used, so let's not create new tag

---

Updated Docs to Documentation.